testcafe/tests/helpers/visualreview-utils.js

```diff
@@ -0,0 +1,46 @@
+import VisualReview from 'visualreview-client'
+import { t } from 'testcafe'
+import { getNavigatorOs, getNavigatorName, getResolution } from './utils'
+
+//Put this const in travis after POC
+const VISUALREVIEW_INSTANCE = 'visualreview.cozycloud.cc'
+
+export class VisualReviewTestcafe extends VisualReview {
+  constructor(options) {
+    super(options)
+    this.options.protocol = 'https'
+    this.options.hostname = VISUALREVIEW_INSTANCE
+  }
+
+  async takeScreenshotAndReview(imageName) {
```

Comment:
The function is called `takeScreenshotAndReview ` but there's no review happening, so I think we should rename it. `saveScreenshot` or something like that?

Reply:
`takeScreenshotAndUpload` ?